qdev_wrappers/plot_functions.py

```diff
@@ -77,15 +77,18 @@ def _create_plot(plot, i, name, data, counter_two, j, k):
                     fulltitle = ""
                 plot.subplots[j + k].set_title(fulltitle)
 
-    for j, i in enumerate(inst_meas):
+    subplot_index = 0
+    for measurement in inst_meas:
```

Comment:
we change i -> measurement.
This is not done for all calls below.

Otherwise this change makes lifeplotting work when calling multiple multi array parameters.